test/AllPairs/test_ap.py

```diff
@@ -4,8 +4,11 @@
 
 import pytest
 
+import src.config as config
 from src.allpairs import AllPairs
 
+config.init_from_file("config/config.yaml")
```

Comment:
The cases cases aren't directly using the config here, but we always need to initialize a config before running a pathway reconstruction algorithm now because that sets up the container framework, right? Adding a comment in at least one of the test cases to explain that may help people reading the code.